cmd/rootMain.go

```diff
@@ -53,6 +53,7 @@ func runShell(cachedir, mountpath string) exitCode {
 		"wexec":    "exec",
 		"find":     "find",
 		"help":     "help",
+		"winfo":    "info",
```

Comment:
It'd be nice if this map were generated by the RootCommand so I don't have to remember to update this whenever I add a new subcommand, but it's not a blocker or that big of a deal.

Reply:
I started doing that, then decided I didn't want to include all of them so left it out.